evalml/tests/model_understanding_tests/test_permutation_importance.py

```diff
@@ -149,6 +152,9 @@ def test_fast_permutation_importance_matches_sklearn_output(mock_supports_fast_i
                     "dependency not installed.")
     X, y = load_fraud(100)
 
+    if pipeline_class == LinearPipelineWithTextFeatures:
+        X = X.set_types(logical_types={'provider': 'NaturalLanguage'})
+
```

Comment:
Just for my edification, is the reason this test covers those lines because the change in logical types forces a derivation of the provenance?

Reply:
Yea the provenance is only computed if the text featurizer creates features. If you run it on a dataset without text features, the provenance will always be an empty dict. Since the text featurizer will (now) only compute features on columns of logical type `NaturalLanguage`, we have to create one for the component to be able to do its thing.